README.md

```diff
@@ -1 +1,142 @@
-# Capstone-1
\ No newline at end of file
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+- [👤 Author](#author)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Capstone #1 <a name="about-project"></a>
+
+'Capstone #1' is the first capstone project in the Microverse program first module, made for learning purposes.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Languages</summary>
+  <ul>
+    <li>HMTL</li>
+    <li>CSS</li>
+    <li>JavaScript</li>
+  </ul>
+</details>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Key Features <a name="key-features"></a>
+
+- **Dynamically generated HTML**
+- **Responsive for both desktop and mobile versions**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+You can checkout the running website here:
+
+- [Live Demo Link](https://jakariajishan.github.io/Capstone-1/)
+
+My presentation of the project in its current state:
+
+- [Presentation](https://www.loom.com/share/375f3bfb04f747a685053d7429744cda)
```

Comment:
- [ ] @JakariaJishan great work on your presentation, however, you forgot to include a section where you highlight some interesting piece of code or something you built that you are very proud of. Also kindly a link to the video in your pull request description, the video should adhere to all the requirements below as previously indicated by the previous reviewer
![image](https://user-images.githubusercontent.com/61408860/212124362-6125f374-970c-4609-a6f6-c3a27042451d.png)